src/sagemaker/tensorflow/__init__.py

```diff
@@ -19,5 +19,12 @@
 # classes for tensorflow serving. Currently tensorflow_serving_api can only be pip-installed for python 2.
 sys.path.append(os.path.dirname(__file__))
 
-from sagemaker.tensorflow.estimator import TensorFlow  # noqa: E402, F401
-from sagemaker.tensorflow.model import TensorFlowModel, TensorFlowPredictor  # noqa: E402, F401
+from sagemaker.tensorflow.estimator import (  # noqa: E402,F401  # pylint: disable=wrong-import-position
+    TensorFlow,
+)
+from sagemaker.tensorflow.model import (  # noqa: E402,F401  # pylint: disable=wrong-import-position
+    TensorFlowModel,
+)
+from sagemaker.tensorflow.model import (  # noqa: E402,F401  # pylint: disable=wrong-import-position
+    TensorFlowPredictor,
+)
```

Comment:
why not:

```python
from sagemaker.tensorflow import model, estimator
```

?

Reply:
That'll work as well, but shouldn't we aim to be more restrictive in our imports?

Reply:
As discussed, per the google python style guide, we should aim to import modules, not classes. Good callout!
Also per the google style guide, I went ahead and made the change in two lines.